tests/data/mc4_clmc4.py

```diff
@@ -0,0 +1,1086 @@
+# coding: utf-8
+#
```

Comment:
If you want to test the cleaning code, you can follow my `core_test.py` and test your patterns if they work as you expected.

For examples, if you want to test your gamble cleaning pattern, you should test your pattern on some gamble text and see if they are removed.

We do this via writing the test function and using pytest module. See more at `core_test.py`